### ISSUE TITLE
Cannot handle struct "values"

### ISSUE THREAD

**Issue by kunos** [SogoCZE/Jails#33]
I am not sure what's the technical name for declaring a structured value like this:

stuff : struct {
   blabla: float;
}

But Jails cannot understand them and can't complete nor navigate to the members;

Here's the simple repro code:
```

stuff : struct {
    stuff_a: float;
    stuff_b: float;
}

StuffStruct :: struct {
    stuff_c: float;
    stuff_d: float;
}

mystuff : StuffStruct;

main :: () {
    print("%\n", mystuff.stuff_c); // It will complete and navigate

    print("%\n", stuff.stuff_a); // It won't complete nor navigate to "stuff_a", it will navigate to "stuff" though

}

```

Environment: VSCode, Jails 0.0.11, Jai 0.1.096

**Comment by kunos** [comment]
Sent a PR (#35) with an easy fix to this.

**Comment by SogoCZE** [comment]
Resolved in #35.
